sleap_nn/inference/paf_grouping.py

```diff
@@ -191,6 +193,11 @@ def make_line_subs(
     )  # (n_candidates, 2, n_line_points)  # dim 1 is [x, y]
     XY = XY[:, [1, 0], :]  # dim 1 is [row, col]
 
+    # clip coordinates for size of pafs tensor.
+    height, width = pafs_hw
+    XY[:, 0, :][XY[:, 0, :] >= height] = height - 1
+    XY[:, 1, :][XY[:, 1, :] >= width] = width - 1
```

Comment:
Much more efficient to do this as a broadcasted ufunc op:

```suggestion
    XY[:, 0] = torch.clip(XY[:, 0], min=0, max=height - 1)
    XY[:, 1] = torch.clip(XY[:, 1], min=0, max=width - 1)
```